snapshot/testutil/helpers.go

```diff
@@ -1,24 +1,13 @@
 package testutil
 
 import (
-	"os/exec"
+	"syscall"
 	"testing"
-
-	"github.com/docker/containerd"
 )
 
-func UnmountAll(t *testing.T, mounts []containerd.Mount) {
-	for _, mount := range mounts {
-		Unmount(t, mount.Target)
-	}
-}
-
 func Unmount(t *testing.T, mountPoint string) {
```

Comment:
These provide failure under error for testing purposes. This can be moved to top-level `testutil` package if it is broadly useful.